.github/workflows/test.yml

```
@@ -32,6 +32,9 @@ jobs:
           - odoo-version: 16.0
             python-version: "3.10"
             machine: ubuntu-22.04
+          - odoo-version: 17.0
+            python-version: "3.10"
+            machine: ubuntu-22.04
```

```suggestion
          - odoo-version: 17.0
            python-version: "3.11"
            machine: ubuntu-22.04
```

- odoo 17 docker image is based on `debian:bookworm`  (https://github.com/odoo/odoo/blob/17.0/setup/package.dfsrc#L3)
- `bookworm` comes with `python3.11` by default (https://wiki.debian.org/Python#Supported_Python_Versions)

Does it makes sense to use 3.11 ?

---

This is for the test of the template itself. It does not matter that much, I think. For for testing the addons we want to test on 3.10 which is the minimum Python version we want to support (in alignment with what Odoo decided in https://github.com/odoo/odoo/pull/136904).